easyContent2Metadata-platform-jar/src/main/java/org/alfresco/extension/ec2m/extractor/PDFExtractor.java

```diff
@@ -20,10 +19,8 @@
  */
 public class PDFExtractor implements Extractor{
 
-    private Logger logger = LoggerFactory.getLogger(this.getClass().getName());
-
     private static final String METADATA_REGION = "metadata";
-
+    private Logger logger = LoggerFactory.getLogger(this.getClass().getName());
```

Comment:
good point. But I think on the long term I haev found a more interesting solution: http://www.yegor256.com/2014/05/23/avoid-java-static-logger.html